src/main/scala/com/codacy/client/bitbucket/v2/Team.scala

```diff
@@ -3,14 +3,13 @@ package com.codacy.client.bitbucket.v2
 import play.api.libs.functional.syntax._
 import play.api.libs.json._
 
-case class Team(uuid: String, username: String, display_name: String)
+case class Team(username: String, display_name: String, uuid: String, avatar: String)
```

Comment:
hum, could we have problems when changing this order? For example in portal if we bump the version or so?

Reply:
I'm thinking that we don't initialize/build this object outside this library?

Reply:
But any reason that we are changing this? I would prefer not to change them if possible. And tbh, I like the order that existed before, seems from "more stable" to "less important"

Reply:
Only reason for the change is this is the order we get back from the API, will revert it :+1: